testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -1493,6 +1493,45 @@ public void getRows() throws IOException {
     assertThat(data.get(0).get("firstname")).isEqualTo("John");
   }
 
+  @Test
+  public void getRowsWithUDT() throws IOException {
+    createKeyspace(keyspaceName);
+
+    // create UDT: note -- UDT names must be lower-case it seems (mixed case fails)
```

Comment:
What is the failure? I'm not aware of a requirement for UDT names to be lower case

Reply:
I'll check this again, I don't remember off-hand. Observed with V1.

Reply:
@jeffreyscarpenter Failure message:

    ERROR [main] 2022-01-10 09:23:37,246 RestUtils.java:279 - {"description":"Bad request: Malformed type name: testUdt","code":400}

which does not occur if `test_udt` is used. I wonder if this is due to added quoting/escaping.